types/datadog-winston/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for datadog-winston 1.0
+// Project: https://https://github.com/itsfadnis/datadog-winston
+// Definitions by: Matt Hintzke <https://github.com/mhintzke13>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+import * as Transport from "winston-transport";
+
+interface DataDogTransportOptions extends Transport.TransportStreamOptions {
```

Comment:
This type cannot be accessed outside of this module. Consider moving into a `DatadogWinston` namespace that merges with the class below so that it can be accessed via imports (similar to `Transport.TransportStreamOptions`).

Also, should it be named `DatadogTransportOptions` to match the class name below?

Reply:
@rbuckton I believe I have made the updates you asked for. Totally makes sense that this wasn't exposed to the consumer prior to my fix. I even updated the test so I explicitly use the type `DatadogWinston.DatadogTransportOptions` just like `Transport.TransportStreamOptions`. Let me know if I did something wrong as I am pretty knew to TS.